[docs] google play in app purchases change (#17396)
expo/expo: docs/pages/versions/v45.0.0/sdk/in-app-purchases.md

### Android

> On Android, you must first create an entry for your app and upload a release APK in the [Google Play Console](https://developer.android.com/distribute/console/). From there, you can configure your in-app purchases and their details under `Monetise > Products > In-app products`.

Reviewer: ```suggestion
On Android, you must first create an entry for your app and upload a release APK in the [Google Play Console](https://developer.android.com/distribute/console/). Then, configure in-app purchases and details under **Monetise > Products > In-app products**.
```
